srw/srw.cpp

```diff
@@ -1027,6 +1112,12 @@ int dnet_cmd_exec(struct dnet_net_state *st, struct dnet_cmd *cmd, const void *p
 	if (!srw)
 		return -ENOTSUP;
 
+	// setup cocaine trace with elliptics trace_id
+	// (cocaine trace_id will be randomly generated if elliptics trace_id is zero)
+	start_or_continue_trace(cmd->trace_id);
```

Comment:
It can, indeed. But tracing support being sporadic among our components we specifically wanted to be able to track request paths through cocaine and back even if a client didn't made an effort to set trace_id.
This wouldn't be necessary (and will go away) when/if elliptics client will automatically use non-empty trace_id by default.